src/app/components/app/HomeLayout/HomeLayout.mobile.jsx

```diff
@@ -45,6 +45,26 @@ const HomeLayout = (props) => {
               );
             }}/>
 
+          <Route
+            path={AppURL.SIGNIN}
+            render={(props) => {
```

Comment:
https://reacttraining.com/react-router/core/api/Route/component
Could you use `component` instead of `render` here?